src/ai/backend/manager/api/vfolder.py

```diff
@@ -491,9 +493,12 @@ async def create(request: web.Request, params: CreateRequestModel) -> web.Respon
                     cast(int, user_row.resource_policy_row.max_vfolder_count),
                     cast(int, user_row.resource_policy_row.max_quota_scope_size),
                 )
+                container_uid = cast(Optional[int], user_row.container_uid)
             case _:
                 raise GroupNotFound(extra_data=group_id_or_name)
 
+        vfolder_permission_mode = 0o775 if container_uid is not None else None
```

Comment:
Is it not possible to use the value of `DEFAULT_VFOLDER_PERMISSION_MODE` here as well?

Reply:
`DEFAULT_VFOLDER_PERMISSION_MODE` is 0o755, not 0o775